### PR TITLE
monop: add page

### PR DESCRIPTION
**Pull request by sbrl**
Following on from my previous 2 pages (#2557 and #2558), I went looking for a .NET assembly browser - and stumbled across this command :P

For my own reference, the Mono project have a [page](https://www.mono-project.com/docs/tools+libraries/tools/) dedicated to the tools that come bundled with `mono`.

----

- [ ] The page (if new), does not already exist in the repo.

- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [ ] The page has 8 or fewer examples.

- [ ] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [ ] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by sbrl** [comment]
No idea :P

This is part of the _Mono Project_ though, which is an open-source implementation of .NET. It's headed up by Xamarin, who were recently bought Microsoft, so Microsoft now own 2 implementations of the same thing..... * shrugs *

**Comment by pxgamer** [comment]
Ah ok. :thinking_face: Not sure why I thought Mono was part of Xamarin now.

File: pages/common/monop.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# monop
+
+> Finds and displays signatures of types and methods inside .NET assemblies.
+
+- Show the structure of a type built-in to the .NET Framework:
```
Comment (agnivade) [inline pages/common/monop.md]: of a built-in type of the .NET framework ? (Should the F be capital or small ?)

Comment (sbrl) [reply pages/common/monop.md]: Thanks! And it should be a capital, as that's it's the name of a thing and the way it appears on their website.

Comment (agnivade) [reply pages/common/monop.md]: I actually wanted _ built-in type of_

Comment (agnivade) [reply pages/common/monop.md]: It just reads natural to me. For eg. "int, and char are built-in types of the C programming language" sounds better than "int and char are types built-in to the C programming language.".

But up to you.

Comment (sbrl) [reply pages/common/monop.md]: I see. Perhaps I should make `type` uppercase then. It's a specific class that holds information about a data-type, and is used in reflection. This command lists all such Types.

Comment (agnivade) [reply pages/common/monop.md]: Sure.

**Pull request merged**